systemtest/src/test/java/io/strimzi/systemtest/kafka/KafkaNodePoolST.java

```diff
@@ -203,6 +205,61 @@ void testKafkaManagementTransferToAndFromKafkaNodePool(ExtensionContext extensio
 
     }
 
+    @ParallelNamespaceTest
+    void testKafkaNodePoolBrokerIdsManagementUsingAnnotations(ExtensionContext extensionContext) {
+
+        final TestStorage testStorage = new TestStorage(extensionContext);
+        final int originalReplicaCount = 2;
+        final int scaledReplicaCount = 5;
+        String nodePoolAnnoIdsWithRange = "[20-21, 99]";
+
+        Kafka kafka = KafkaTemplates.kafkaPersistent(testStorage.getClusterName(), originalReplicaCount, 1)
+            .editOrNewMetadata()
+                .addToAnnotations(Annotations.ANNO_STRIMZI_IO_NODE_POOLS, "enabled")
+            .endMetadata()
+            .build();
+
+        KafkaNodePool kafkaNodePool =  KafkaNodePoolTemplates.defaultKafkaNodePool(testStorage.getKafkaNodePoolName(), testStorage.getClusterName(), originalReplicaCount)
+            .editOrNewMetadata()
+                .withNamespace(testStorage.getNamespaceName())
+            .endMetadata()
+            .editOrNewSpec()
+                .addToRoles(ProcessRoles.BROKER)
+                .withStorage(kafka.getSpec().getKafka().getStorage())
+                .withJvmOptions(kafka.getSpec().getKafka().getJvmOptions())
+                .withResources(kafka.getSpec().getKafka().getResources())
+            .endSpec()
+            .build();
+
+        resourceManager.createResourceWithWait(extensionContext,
+            kafkaNodePool,
+            kafka);
+
+        // Save for later comparison
+        List<Integer> originalNodePoolIds = KafkaNodePoolUtils.getCurrentKafkaNodePoolIds(testStorage.getNamespaceName(), testStorage.getKafkaNodePoolName());
+        List<Integer> expectedNodePoolIds = KafkaNodePoolUtils.parseNodePoolIdsWithRangeStringToIntegerList(nodePoolAnnoIdsWithRange);
```

Comment:
Why don't you hardcode this instead of having such a complicated method? You already hardcoded the range from which the node IDs will be picked. So I do not think it would matter if you hardcode this as well.

Reply:
Yes, i will change that, thank you